src/EnergyPlus/SizingManager.cc

```diff
@@ -1823,30 +1820,28 @@ void ManageSystemVentilationAdjustments(EnergyPlusData &state)
             } else {
                 numZones = AirToZoneNodeInfo(AirLoopNum).NumZonesHeated;
             }
-            for (int zoneNum = 1; zoneNum <= numZones; ++zoneNum) {
+            for (int airLoopZoneNum = 1; airLoopZoneNum <= numZones; ++airLoopZoneNum) {
```

Comment:
Changed name of loop counter to `airLoopZoneNum`. `ZoneNum` was not the real zone number, but rather the counter for heated or cooled zones on a given airloop. But `ZoneNum` was used below to (incorrectly) access zone multipliers.